fab/environments.yml

```diff
@@ -179,8 +179,12 @@ icds:
         num_processes: 8
       kafka-ucr-static-ccs_record_cases_monthly:
         num_processes: 8
+      kafka-ucr-static-ccs_record_cases_monthly_v2:
```

Comment:
It'll probably be nicer to have one pillow kafka-ucr-static-test for this. You can scale the number of processes if necessary. The system page is really long these days